dts/bindings/gpio/launchxl-connector.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2022 Dhruva Gole
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    GPIO pins exposed on SimpleLink Wi-Fi CC3220SF LaunchPad development kit headers.
```

Comment:
Isn't this `launchxl-connector.yaml` supposed to be common across all LAUNCHXL series boards?

Reply:
But okay, the connector.yaml should probably stay general, so I will renove the specific part name

Reply:
But in that case we already have a yaml for that : https://docs.zephyrproject.org/latest/build/dts/api/bindings/gpio/ti,boosterpack-header.html

Reply:
But, again as I said the GPIO NUM mapping to the physical headers are different atleast in the case of CC32x seems to be...
for eg. Physical pin 27, 28 here are not analog, but in boosterpack 27, 28 are marked as Analog.